BorrowToOwn.API/Controllers/BrandController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using BorrowToOwn.Services.Communications.RequestObject.DTO;
+using BorrowToOwn.Services.Contracts;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BorrowToOwn.API.Controllers
+{
+    [ApiController]
+    [Route("/api/Brands")]
+    public class BrandController : Controller
+    {
+        private readonly IBrandService _brandService;
+
+        public BrandController(IBrandService brandService)
+        {
+            _brandService = brandService ?? throw new ArgumentNullException(nameof(brandService));
+        }
+        // GET: /<controller>/
+        [HttpGet("")]
+        public async Task<IActionResult> Get()
+        {
+            var cats = await _brandService.GetBrandsAsync();
+
+            return Ok(cats);
+        }
+
+        [HttpGet("{id:int}", Name = "GetBrand")]
+        public async Task<IActionResult> GetBrand(int id)
+        {
+            var cat = await _brandService.GetBrandAsync(id);
+            if (cat == null) return NotFound();
```

Comment:
let me do like your oyinbo friends.... ternary operator will be nice here.

Reply:
yeah ... thought that too ... let me fix that